resources/META-INF/plugin.xml

```diff
@@ -615,7 +615,7 @@ miscellaneous:
     <projectService serviceInterface="io.flutter.view.FlutterView"
                     serviceImplementation="io.flutter.view.FlutterView"
                     overrides="false"/>
-    <projectOpenProcessor implementation="io.flutter.project.FlutterProjectOpenProcessor" order="first"/>
+    <projectOpenProcessor id="flutter" implementation="io.flutter.project.FlutterProjectOpenProcessor" order="first"/>
```

Comment:
Is this id unique enough? I'm not sure what other ids in this space would look like.

Reply:
None of the others have id's, so I think this is good. It is a very small name-space